app/assets/stylesheets/regions/mastheads/boxes-masthead.scss

```diff
@@ -227,47 +227,4 @@
     min-height: 340px !important;
     padding-bottom: 0;
   }
-}
-
```

Comment:
Note: I deleted `app/assets/stylesheets/widgets/masthead.scss` and created `app/assets/stylesheets/widgets/boxes-masthead.scss`. I'm not sure why git is making it seem like i just moved the file.